setup.cfg

```diff
@@ -41,4 +41,6 @@ install_requires =
 test = codecov; ipykernel; pytest>=5.3.2; pytest-cov; jupyter_server[test]; openapi_core~=0.14.0; pytest-console-scripts; strict-rfc3339; ruamel.yaml; wheel
 
 [options.packages.find]
-exclude = ['docs*']
+exclude =
+    docs*
+    tests*
```

Comment:
@blink1073 I think this is responsible for [this error](https://github.com/jupyterlab/jupyterlab/runs/5571001829?check_suite_focus=true):
```
ImportError: Error importing plugin "jupyterlab_server.pytest_plugin": No module named 'jupyterlab_server.pytest_plugin'
```

Reply:
Good point, I'll make a fix